azure-pipelines.yml

```diff
@@ -0,0 +1,65 @@
+trigger:
+- Dev
+
+pool:
+  vmImage: ubuntu-latest
+
+variables:
+  microservice: tracking-microservice
+  coveragePath: src/main/java/com/revature/registry/aspect,controller,model,repository,service
+
+steps:
+- checkout: self
+
+- task: SonarCloudPrepare@1
+  inputs:
+    SonarCloud: 'SonarCloud'
+    organization: '1053-august-duet-project-registry'
+    scannerMode: 'Other'
+    extraProperties: |
+      # Additional properties that will be passed to the scanner, 
+      # Put one key=value per line, example:
+      # sonar.exclusions=**/*.bin
+      sonar.projectKey=1053-August-Duet-Project-Registry_project-registry-${{ variables.microservice }}
```

Comment:
things that would change on the next iteration of this project, like what github org, what sonarcloud org, sonar key, etc; should all be kept in one place like variables.